src/Remove.js

```diff
@@ -0,0 +1,102 @@
+import { getStatus, showItems } from './List.js';
+
+let textFields = document.querySelectorAll('.todo');
+let dotsIcon = document.querySelectorAll('.fa-ellipsis-v');
+let trashIcon = document.querySelectorAll('.fa-trash-alt');
+let allDivs = document.querySelectorAll('.todo-item');
+let label = document.querySelectorAll('.label');
+let checkBtn = document.querySelectorAll('.index');
+let checkMark = document.querySelectorAll('.fa-check');
+let fieldId;
+
+function getCurrent() {
+    textFields = document.querySelectorAll('.todo');
+    dotsIcon = document.querySelectorAll('.fa-ellipsis-v');
+    trashIcon = document.querySelectorAll('.fa-trash-alt');
+    allDivs = document.querySelectorAll('.todo-item');
+    label = document.querySelectorAll('.label');
+    checkBtn = document.querySelectorAll('.fa-square');
+    checkMark = document.querySelectorAll('.fa-check');
+}
+
+function sortToDo(fieldId) {
+    getCurrent();
+    const unsortedItems = getStatus();
+    for (let i = fieldId; i < unsortedItems.length; i += 1) {
+        unsortedItems[i].index = i;
+        allDivs[i].setAttribute('data-id', i);
+        checkBtn[i].setAttribute('data-id', i);
+        checkMark[i].setAttribute('data-id', i);
+        dotsIcon[i].setAttribute('data-id', i);
+        trashIcon[i].setAttribute('data-id', i);
+        label[i].setAttribute('for', i);
+        textFields[i].setAttribute('id', i);
+    }
+    localStorage.setItem('toDoList', JSON.stringify(unsortedItems));
+    getStatus();
+}
```

Comment:
- [ ] By default the property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). Currently, your base index is `0`. Let's fix that